alf/algorithms/config.py

```diff
@@ -65,6 +66,12 @@ def __init__(self,
             unroll_length (int):  number of time steps each environment proceeds per
                 iteration. The total number of time steps from all environments per
                 iteration can be computed as: ``num_envs * env_batch_size * unroll_length``.
+            unroll_with_grad (bool): a bool flag indicating whether we require
+                grad during ``unroll()``. ``OnPolicyAlgorithm`` always unrolls with
+                grads. Thus this flag is only used by ``OffPolicyAlgorithm``. In
```

Comment:
Suggest to move ```This flag is only used by OffPolicyAlgorithm ...enable this flag for its training``` immediately after ```we require
                grad during unroll()```. And move The explanation to ```OnPolicyAlgorithm``` case to the end.

Reply:
updated